### ISSUE TITLE
Withings authentication error

### ISSUE THREAD

**Issue by charlesaberger** [jaroslawhartman/withings-sync#4]
Hi,

Trying to set this up for the first time.

Using the URL provided by the app (https://account.withings.com/oauth2_user/authorize2?response_type=code&client_id=183e03e1f363110b3551f96765c98c10e8f1aa647a37067a1cb64bbbaf491626&state=OK&scope=user.metrics&redirect_uri=https%3A%2F%2Fwieloryb.uk.to%2Fwithings%2Fwithings.html&selecteduser=22217232) I get the following error after logging in and allowing this app:

```Json
{"errors":[{"message":"redirect_uri_mismatch: The redirect URI provided is missing or does not match partner callback url"}]}
```

Am I doing something wrong?

Thanks.

**Comment by jaroslawhartman** [comment]
Hi Charles,

Thanks for reporting; we're in process of migrating the repo and perhaps there are some leftover from previous setup.

From where you got the script? As a quick workaround - please update file `withings_sync/config/withings_app.json` as per https://github.com/jaroslawhartman/withings-sync/blob/master/withings_sync/config/withings_app.json , i.e.

```
{
    "callback_url": "https://jaroslawhartman.github.io/withings-sync/contrib/withings.html",
    "client_id": "183e03e1f363110b3551f96765c98c10e8f1aa647a37067a1cb64bbbaf491626",
    "consumer_secret": "a75d655c985d9e6391df1514c16719ef7bd69fa7c5d3fd0eac2e2b0ed48f1765"
}
```

Regards,
Jarek

**Comment by jaroslawhartman** [comment]
Hi @stv0g 

Assigning to you to update Docker images and PyPi package - this was missing after updating Callback URL to Github pages. Perhaps we could also setup an automatic workflow?

Regards,
Jarek

**Comment by charlesaberger** [comment]
> From where you got the script?

I installed it as per the instructions in the README.md.

> As a quick workaround - please update file `withings_sync/config/withings_app.json` as per https://github.com/jaroslawhartman/withings-sync/blob/master/withings_sync/config/withings_app.json , i.e.
> 

Thanks - it seems to be working now as I am able to get a token now.

**Comment by jaroslawhartman** [comment]
Great, thank you for confirming. Steffen will update PyPI files at next opportunity.

**Comment by stv0g** [comment]
Hi @jaroslawhartman @charlesaberger 

I published a new release v3.1.2 on PyPi.org and Docker Hub.
